### PR TITLE
162633516 Ft report article

### PR DESCRIPTION
**Pull request by Qadriyah**
#### What does this PR do?
* Enables users to report articles that violate the terms of the agreement or are plagiarized.
* Enables the admin to view the reported articles
#### Tasks to be completed
* Report article
* View reported articles
* Write test cases
#### How should this be manually tested
* Run migrations
```
python manage.py migrate
```
* Start server
```
python manage.py runserver
```
* Running tests
```
python manage.py test authors/apps/articles/tests/test_report_articles.py
```
* Endpoint to be tested
```
POST /api/v1/articles/slug/report
GET /api/v1/articles/slug/report
GET /api/v1/articles/reported
```
* Header
```
Content-Type: application/json
Authorization: Get JWT Token from login/register endpoints
```
* Post data
```
{
	"message": "This post violates the terms and conditions of Author's Haven"
}
```
#### Relevant PT stories
[#162633516](https://www.pivotaltracker.com/n/projects/2231917/stories/162633516)

### CODE REVIEW

**Comment by mubstimor** [comment]
LGTM

**Comment by mycok** [comment]
LGTM

**Comment by mycok** [comment]
LGTM

File: authors/settings.py [hunk 1]
```diff
@@ -209,3 +221,21 @@
...
+NOSE_ARGS = [
+    '--with-coverage',
+    '--cover-erase',
+    '--cover-inclusive',
+    '--cover-package=authors',
+    '--verbosity=3'
+]
+
+# Email Settings
+EMAIL_HOST = config('EMAIL_HOST')
```
Comment (lym) [inline authors/settings.py]: It appears these environment variables are already in the develop branch

File: authors/apps/authentication/views.py [hunk 2]
```diff
@@ -65,6 +69,46 @@ def send_verification_email(self, user, request):
         return email_template(subject, content, user.email)
 
 
+class UserAccountVerificationAPIView(APIView):
```
Comment (lym) [inline authors/apps/authentication/views.py]: This class is already in develop too.

**Comment by jeanjoe** [comment]
LGTM

**Pull request merged**